modules/persistenceService/src/main/scala/connectionLayer/DbQueries.scala

```diff
@@ -0,0 +1,35 @@
+package connectionLayer
+
+import persistenceAlgebra.DbAlgebra
+import persistenceModel.{Id, Name, User, UserName}
+import doobie._
+import doobie.implicits._
+import cats.effect.Bracket
+import doobie.postgres.implicits._
+
+import java.util.UUID
+
+class DbQueries[F[_]](implicit ev: Bracket[F, Throwable]) extends DbAlgebra {
+  def insert(user: User): ConnectionIO[Int] = DbQueries.insert(user).run
+
+  def find(username: UserName): ConnectionIO[Option[User]] = DbQueries.find(username).option
+
+  def remove(user: UserName): ConnectionIO[Int] = DbQueries.remove(user).run
+}
+
+object DbQueries {
+
+  implicit val nameMeta         = Meta.StringMeta.timap[Name](Name(_))(name => name.name)
+  implicit val userName         = Meta[String].timap[UserName](UserName(_))(userName => userName.username)
+  implicit val idMeta: Meta[Id] = Meta[UUID].timap(uuid => Id(uuid))(id => id.id)
+
+  def insert(user: User): Update0 =
+    sql"insert into userdb (name) values (${user.name})".update
```

Comment:
I guess we want to insert the three fields, right?
```suggestion
    sql"insert into userdb (id, name, username) values (${user.id}, ${user.name}, ${user.username})".update
```